src/python/clawutil/test.py

```diff
@@ -155,6 +156,9 @@ def setUp(self):
         self.stdout.write("  %s" % self.temp_path)
         self.stdout.write("  %s" % self.test_path)
         self.stdout.flush()
+        # clean up *.o files in test path
+        for path in glob.glob(os.path.join(self.test_path,"*.o")):
```

Comment:
Should probably add `*.mod` to this. Also, does this work if the glob returns nothing?